src/Compat.jl

```diff
@@ -65,6 +65,15 @@ if VERSION < v"1.2.0-DEV.272"
     export hasproperty
 end
 
+if VERSION < v"1.3.0-alpha"
```

Comment:
```suggestion
if VERSION < v"1.3.0-DEV.349"
```
Not that it matters much, but we generally try to use the exact version.

Reply:
Sure. Thanks for the suggestion !
BTW how do you figure out this version number.

Reply:
I have just added tests as you suggested.